source/initial-dns-seedlist-discovery/tests/replica-set/dbname-with-commas-escaped.yml

```diff
@@ -0,0 +1,13 @@
+uri: "mongodb+srv://test1.test.build.10gen.cc/some%2Cdb?replicaSet=repl0"
+seeds:
+  - test1.test.build.10gen.cc:27017
+  - test1.test.build.10gen.cc:27018
```

Comment:
This needs to be:
```
seeds:
    - localhost.test.build.10gen.cc:27017
    - localhost.test.build.10gen.cc:27018
```

Reply:
Yeah nice catch - looks like our assertions in node are incorrect.

Reply:
Updated.